io/private/EptSupport.hpp

```diff
@@ -154,6 +156,7 @@ class PDAL_DLL EptInfo
     const Json::Value m_info;
     BOX3D m_bounds;
     uint64_t m_points = 0;
+    uint64_t m_span = 0;
```

Comment:
The other items seem obvious.  This doesn't.  What is it, exactly?

Reply:
Span is the voxel span in each direction for the octree nodes.  A dataset whose octree nodes are 256x256x256 would have a span of `256`.  See https://github.com/connormanning/entwine/blob/master/doc/source/entwine-point-tile.md#span.

Reply:
Maybe just a comment that says something like "length, width and depth of each octree node in voxels"  Or a link to the doc?

Reply:
Done at 604d1006e250dd183a76a16eb6c6ed98041f5077